internal/ethapi/api.go

```diff
@@ -52,6 +53,10 @@ import (
 )
 
 var errBlobTxNotSupported = errors.New("signing blob transactions not supported")
+var (
+	gasUsedEthEstimateGasGauge = metrics.NewRegisteredGauge("api/eth_call/gas", nil)
+	gasUsedEthCallGauge        = metrics.NewRegisteredGauge("api/eth_estimateGas/gas", nil)
```

Comment:
how about something like `rpc/gas_used/eth_call`?  Seems to fit in better to existing metrics.

Metric is only ever incremented, so should be a counter instead of a gauge.

Reply:
Done.